tests/view_tests/tests/test_i18n.py

```diff
@@ -41,8 +44,15 @@ def test_setlang(self):
         post_data = dict(language=lang_code, next='/')
         response = self.client.post('/i18n/setlang/', post_data, HTTP_REFERER='/i_should_not_be_used/')
         self.assertRedirects(response, '/')
+        # Session assertion removed in Django 2.1
```

Comment:
If line length isn't an issue...
`self.assertEqual(self.client.session[LANGUAGE_SESSION_KEY], lang_code)  # RemovedInDjango21Warning`
or something like "The following assertion is RemovedInDjango21Warning"